### PR TITLE
Mobile setup

### PR DESCRIPTION
**Pull request by kennedyowusu**
1. Created a new branch with the name mobile_setup
2. Implemented the HTML and CSS for the mobile version of the home page from scratch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,12 +80,14 @@ <h1 class="title">Hey There. I'm Kennedy</h1>
...
+
+      <section>
         <img
-          class="bg-image"
+          class="bg-img"
           src="images/group-bg-img.png"
           alt="Background Image"
-          width="261"
-          height="256"
+          srcset=""
```
Comment (Olamarx) [inline index.html:90]: - [ ] Kindly add this as a background image through the CSS. Kindly check [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) to know how when to use the image and CSS background. Learn how to add the background [here](https://www.w3schools.com/cssref/pr_background-image.asp)

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @kennedyowusu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



- [ ] Kindly stick to the figma template in terms of font size, width, color and others.
- [ ] The image should be added as background through CSS. Check the inline comment, please.
## Your Design
image
## Figma Design
image


 ## Highlights
- Meaningful commit messages ✔️ 
- All Linters Passed ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -5,77 +5,92 @@
...
+                  src="images/medium.png"
+                  alt="Medium Logo"
+              /></a>
+            </li>
+
+            <img
+              src="images/group-bg-img.png"
+              alt="Background Image"
+              class="hero-image"
+            />
```
Comment (vickymarz) [inline index.html:91]: - [ ] As the previous reviewer has suggested, Kindly add this as a background image through the CSS. Kindly check [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) to know how when to use the image and CSS background. Learn how to add the background [here](https://www.w3schools.com/cssref/pr_background-image.asp)

File: index.html [hunk 3]
```diff
@@ -5,77 +5,92 @@
...
-        </ul>
-        <img
-          class="bg-image"
-          src="images/group-bg-img.png"
-          alt="Background Image"
-          width="261"
-          height="256"
-        />
+        <div class="outer-wrapper">
+          <div class="inner-wrapper">
```
Comment (vickymarz) [inline index.html:41]: - [ ] To wrap a list of items, you should use `<ul>` together with` <li>` and not `<div class="inner-wrapper">`

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @kennedyowusu, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @kennedyowusu 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**